index.html

```diff
@@ -11,18 +11,268 @@
     </head>
     <body>
         <!-- header -->
+        <header class="container bg-main">
+            <div class="row">
+                <div class="col-12">
+                    CONTACTS + SOCIALS
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-12">
+                    LOGO + NAV + BASKET, SEARCH ITEMS
+                </div>
+            </div>
+        </header>
+
+    <main>
         <!-- hero -->
+        <section class="container">
+            <div class="row">
+                <div class="col-12 col-md-6 col-lg-6">
+                    <h1>hero</h1>
+                    <p>paragraph</p>
+                    <div class="btn">button</div>
+                    <div class="btn">button</div>
+                </div>
+                    <img src="" alt="robot" class="hidden visible-md col-md-6 col-lg-3 ">
+                    <img src="" alt="mechanics" class="hidden visible-lg col-lg-3 ">
+            </div>
+        </section>
+
         <!-- introduction -->
+        <section class="container">
+            <div class="row">
+                <div class="col-12 col-lg-6">
+                        <div class="column">content</div>
+                        <div class="column">content</div>
+                        <div class="column">content</div>
+                        <div class="column">content</div>
+                </div>
+                <div class="col-12 col-lg-6">
+                    <h2>introduction</h2>
+                    <p>paragraph</p>
+                    <div class="btn">button</div>
+                </div>
+            </div>
+        </section>
+
         <!-- resume -->
+        <section class="container">
+            <div class="row">
+                <div class="col-12 col-lg-6">
+                    <h2>resume</h2>
+                    <p>paragraph</p>
+                    <div class="btn">button</div>
+                    <div class="btn">button</div>
+                </div>
+                <img src="" alt="resume-img" class="col-12 col-lg-5 m-l-lg-1">
+            </div>
+        </section>
+
         <!-- achievements -->
+        <section class="container bg-main">
+            <div class="row">
+                <div class="col-12 col-md-6 col-log-3">
+                    achievements
+                </div>
+                <div class="col-12 col-md-6 col-log-3">
+                    achievements
+                </div>
+                <div class="col-12 col-md-6 col-log-3">
+                    achievements
+                </div>
+                <div class="col-12 col-md-6 col-log-3">
+                    achievements
```

Comment:
cia reikia pataisyti kad desktop versijoj achievement uzimtu 3 stulpelius (col-lg-3)

Reply:
Dėkui, kad pastebėjai, jau pavargęs buvau, pražiobsojau. Vietoj algoritmų pradėjau logaritmus rašyti :D . Kaip išdėstymas ? Ar logiškas ?

Reply:
ten prie mobile versijos kazkaip su logo reiks suzaist, nes jis nubega is hero headeri...

Reply:
dar submit now forma nerespondina prie maziausios versijos

Reply:
liuks atrodo... jau noro yra kazka daryt :)

Reply:
> dar submit now forma nerespondina prie maziausios versijos

Ačiū už pastabas, matot, kiek praleidau :D Gal dar kažką rasit :)

Reply:
manau kad galima mergint, paskui vistiek daugiau matysis
tiesiog gal dar situos pakeitimus padaryk ir ok bus

Reply:
Forma pataisiai, o kas su logo headeriu ? Nelabai supratau

Reply:
logo turi sokinet is viraus i apacia kai keiti ekrano ploto. cia gal grido reikia? nzn...

Reply:
Paziurek originalia versija, ten logo nenusoko :)